builder/builder-context.c

```diff
@@ -543,6 +548,12 @@ builder_context_enable_rofiles (BuilderContext *self,
   if (!self->use_rofiles)
     return TRUE;
 
+  if (!self->have_rofiles)
+    {
+      g_debug ("rofiles-fuse not available, doing without");
```

Comment:
I think this should be g_warning, because even if this *works* it is not the experience we want by default.